server/middleware/cleanLocks.js

```diff
@@ -0,0 +1,48 @@
+const Cohort = require('../models/cohort.model');
+const List = require('../models/list.model');
+
+const unlockLocks = () => {
+  List.updateMany(
+    { locks: { $exists: true } },
+    {
+      locks: { name: false, description: false }
+    }
+  ).catch(() => {
+    // eslint-disable-next-line no-console
+    console.log('Unlocking locks failed...');
```

Comment:
Please mark specific collection that failed, for example with `console.error('Unlocking lists locks failed')`.